opm/parser/eclipse/EclipseState/Schedule/Group.hpp

```diff
@@ -51,6 +51,7 @@ namespace Opm {
         bool isProductionGroup(size_t timeStep) const;
         bool isInjectionGroup(size_t timeStep) const;
         void setProductionGroup(size_t timeStep, bool isProductionGroup);
+        void setInjectionGroup(size_t timeStep, bool isInjectionGroup_);
```

Comment:
The fact that a group can be both an injection group and a production group at the same time is so special that even I would like a comment😀.